<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PadMapping</class>
 <widget class="QWidget" name="PadMapping">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pad Mapping</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="rotationLabel">
       <property name="text">
        <string comment="Rotation of the tablet pad">Rotation:</string>
       </property>
       <property name="buddy">
        <cstring>rotationComboBox</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="KComboBox" name="rotationComboBox">
       <property name="toolTip">
        <string>Select the rotation of the tablet device.</string>
       </property>
       <item>
        <property name="text">
         <string comment="no pad rotation">none</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string comment="rotate the pad counter clock wise">couter clockwise</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string comment="rotate the tablet clockwise">clockwise</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string comment="pad rotation up side down (180Â°)">half</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="xrandrRotationCheckBox">
       <property name="toolTip">
        <string>If enabled the tablet is automatically rotated together with the screen</string>
       </property>
       <property name="text">
        <string>Rotate tablet with screen</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelScreenN0">
       <property name="text">
        <string>Map to output:</string>
       </property>
       <property name="buddy">
        <cstring>screenComboBox</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="KComboBox" name="screenComboBox">
       <property name="toolTip">
        <string>Select the XRandR display which will be used to map the tablet area on it.</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="workingAreaBox">
     <property name="toolTip">
      <string>If enabled, the working area can be changed.
 If disabled the default values from xorg are used</string>
     </property>
     <property name="title">
      <string>Adjust Working Area</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="labelAlignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <item row="0" column="0">
       <widget class="KPushButton" name="calibrateButton">
        <property name="toolTip">
         <string>Opens a fullscreen window to calibrate the tablet.</string>
        </property>
        <property name="text">
         <string>Calibrate Tablet</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>161</width>
          <height>147</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="KIntSpinBox" name="topX">
          <property name="toolTip">
           <string>X-Coordinate of the top left point</string>
          </property>
          <property name="maximum">
           <number>99999</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="KIntSpinBox" name="topY">
          <property name="toolTip">
           <string>Y-Coordinate of the top right point</string>
          </property>
          <property name="maximum">
           <number>99999</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="3">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="3">
         <widget class="KIntSpinBox" name="bottomY">
          <property name="toolTip">
           <string>Height of the used tablet area</string>
          </property>
          <property name="maximum">
           <number>99999</number>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="2">
         <widget class="KIntSpinBox" name="bottomX">
          <property name="toolTip">
           <string>Width of the tablet area</string>
          </property>
          <property name="maximum">
           <number>99999</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1" rowspan="3" colspan="2">
         <widget class="QLabel" name="label">
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="text">
           <string>Working
Area</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>rotationComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>PadMapping</receiver>
   <slot>profileChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>topY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>PadMapping</receiver>
   <slot>profileChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bottomY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>PadMapping</receiver>
   <slot>profileChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bottomX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>PadMapping</receiver>
   <slot>profileChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>87</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>topX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>PadMapping</receiver>
   <slot>profileChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>40</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>calibrateButton</sender>
   <signal>clicked()</signal>
   <receiver>PadMapping</receiver>
   <slot>showCalibrationDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>84</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>screenComboBox</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>PadMapping</receiver>
   <slot>profileChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>workingAreaBox</sender>
   <signal>clicked()</signal>
   <receiver>PadMapping</receiver>
   <slot>profileChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>450</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xrandrRotationCheckBox</sender>
   <signal>clicked()</signal>
   <receiver>PadMapping</receiver>
   <slot>profileChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>452</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>profileChanged()</slot>
  <slot>showCalibrationDialog()</slot>
 </slots>
</ui>
